lib/util/trip/pending_ride_card.dart

```diff
@@ -126,7 +127,7 @@ class PendingRideCard extends TripCard<Ride> {
               ),
               Padding(
                 padding: const EdgeInsets.all(8.0),
-                child: Text("Price: ${trip.price.toString()} €"),
+                child: Text("${AppLocalizations.of(context)!.price}: ${trip.price.toString()} €"),
```

Comment:
price should be an argument of the translation instead, because in some languages the order might be different

Reply:
yes thanks for the hin